WordPress/Classes/ViewRelated/Me/All Domains/Views/DomainPurchaseChoicesView.swift

```diff
@@ -89,12 +89,12 @@ struct DomainPurchaseChoicesView: View {
                     action()
                 }
                 .padding(.bottom, Length.Padding.double)
+                .disabled(viewModel.isGetDomainLoading)
```

Comment:
I added `disable(isLoading)` to `PrimaryButton` itself. But I added it here too, to disable both buttons. As the second button shouldn't care or know anything about the first button and I didn't want to add a second boolean and convolute the PrimaryButton API.